submissions/Temu4/tiny-js-world/index.js

```diff
@@ -0,0 +1,62 @@
+// ======== OBJECTS DEFINITIONS ========
+const man = {
+  species: 'human',
+  gender: 'male',
+  name: 'John',
+  age: 28,
+  hands: 2,
+  legs: 2,
+  hobby: 'fishing',
+  speach: 'Hello, people!'
+};
+
+const woman = {
+  species: 'human',
+  gender: 'female',
+  name: 'Anny',
+  age: 23,
+  hands: 2,
+  legs: 2,
+  hobby: 'cycling',
+  speach: 'Hello, everybody!'
+};
+
+const dog = {
+  species: 'dog',
+  gender: 'male',
+  name: 'Rex',
+  age: 3,
+  hands: 0,
+  legs: 4,
+  hobby: 'diging bones',
+  speach: 'Wohw!'
+};
+
+const cat = {
+  species: 'cat',
+  gender: 'female',
+  name: 'Sida',
+  age: 1.5,
+  hands: 0,
+  legs: 4,
+  hobby: 'sleaping',
+  speach: 'Meow!'
+};
+
+//Create message from object properties
+const createMessage = obj => {
+  let msg1 = `\t${obj.speach} This is <strong>${obj.name}</strong>. `;
+  let msg2 = ` is a ${obj.species} (with ${obj.hands} hands and ${
```

Comment:
IMO this variables are redudant. Let's improve you skills and return `array` with delimiter `join()`.

Reply:
There are rather complicated output message, so variables a also not so simply. I think, greater separation of responsibilities in variables makes the code overweight. The output consist of 3 sentences, And
```javascript
let msg1, msg2, msg3
```
responsible for this sentences.

Reply:
@Temu4  Please, change this function so the output according to [technicaly task](https://github.com/OleksiyRudenko/a-tiny-JS-world#the-job). This can change you compicated output message to simplier. You can use `Array` methods for this. Let me know when you complete.

Reply:
@Temu4  But If you really want add some text for your output - it must be looks like following.
```javascript
return [
        'some text',
        `some text with ${variable}`,
        obj.gender === 'male' ? 'he' : 'she',
    ].join(' ');
```
This approach make you code simplier for understand without `msg1`, `msg2`, `msg3` etc. variables.

Reply:
fix in [this](https://github.com/kottans/frontend-2019-p2p/pull/44/commits/38702b7912088fa4d925c81da09dada474a2adbf) commit